rasa/nlu/utils/spacy_utils.py

```diff
@@ -129,18 +130,96 @@ def get_text(self, example, attribute):
 
         return self.preprocess_text(example.get(attribute))
 
+    @staticmethod
+    def merge_content_lists(
+        indexed_training_samples: List[Tuple[int, Text]], doc_lists: List[Tuple[int, "Doc"]]
+    ) -> List[Tuple[int,"Doc"]]:
+        """Merge lists with processed Docs back into their original order."""
+
+        dct = dict(indexed_training_samples)
+        dct.update(dict(doc_lists))
+        return list(sorted(dct.items()))
```

Comment:
Don't need `list` conversion, since sorted will return a `list`
```suggestion
        return sorted(dct.items())
```

Reply:
Fixed in 8ab6e87